.github/workflows/ci-openapi.yml

```
@@ -138,10 +140,11 @@ jobs:
 
             No changes to OpenAPI specification found. See history of this comment for previous changes.
 
-  publish:
+  publish_unstable:
```

```suggestion
  publish-unstable:
```

our workflows use dashes for job names